scripts/jenkins/build

```diff
@@ -57,8 +79,11 @@ fi
 
 if [[ $TRANSPORT == "lnet" ]]; then
 	PACKAGE='lustre-client'
-else
+elif [[ $TRANSPORT == "libfabric" ]]; then
 	PACKAGE='libfabric'
+else
```

Comment:
Also add a check for other motr transports like `sock`, `membulk`.

The `sock` and `membulk` transports do not require any package installation and hence the installation step can be skipped for these transports, but need to use the below option during motr code compilation.

`CONFIGURE_OPTS=--with-default-network=<SOCK|LNET|LF|MEMBULK>`

Reply:
Done.

I have still kept the installation of package as it is, because even if you use the above option in configure it still requires the packages. But the advantage is even if you have multiple packages installed using the above option should use the appropriate transport.